src/components/CasesCard/index.js

```diff
@@ -27,19 +28,19 @@ const Severe = styled(Tile)`
   border-bottom: ${(props) => (props.selected ? '0.625rem' : '0.3125rem')} solid;
   border-color: ${(props) =>
     props.riskType === 'high'
-      ? '#eb2f2f'
+      ? COLOR_CODE.highRisk
```

Comment:
@Shreets please move code below to the utilities and reuse everywhere
`(props) =>
    props.risk === 'High'
      ? COLOR_CODE.highRisk
      : props.risk === 'Moderate'
      ? COLOR_CODE.moderateRisk
      : props.risk === 'Discharged'
      ? COLOR_CODE.dischargedRisk
      : props.risk === 'Uncertain'
      ? COLOR_CODE.uncertainRisk
      : COLOR_CODE.mildRisk`

Reply:
crated utils function `getColorCode()` to fetch colors for risk status